specification/attestation/resource-manager/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -0,0 +1,614 @@
+        },
+        "status": {
+          "type": "string",
+          "description": "Status of attestation service.",
+          "enum": [
+            "Ready",
+            "NotReady",
+            "Error" 
+          ],
+          "x-ms-enum": {
+            "name": "AttestationServiceStatus",
+            "modelAsString": true
+          }
+        },
+        "attestUri": {
+          "type": "string",
+          "description": "Gets the uri of attestation service"
+        }
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "description": "An error response from Attestation."
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for displaying in a user interface."
+        }
+      },
+      "description": "An error response from Attestation."
+    },
+    "AttestationProviderListResult": {
+      "description": "Attestation Providers List.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Attestation Provider array.",
+          "items": {
+            "$ref": "#/definitions/AttestationProvider"
+          }
+        }
+      }
+    },
+    "AttestationServicePatchParams": {
+      "description": "Parameters for patching an attestation service instance",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the attestation service instance."
+        }
+      }
+    },
+    "AttestationServiceCreationParams": {
+      "description": "Parameters for creating an attestation service instance",
+      "required": [
+        "location",
+        "properties" 
+      ],
+      "x-ms-azure-resource": true,
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The supported Azure location where the attestation service instance should be created."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the attestation service instance."
+        },
+        "properties": {
+          "$ref": "#/definitions/AttestationServiceCreationSpecificParams",
+          "description": "Properties of the attestation service instance"
+        }
+      }
+    },
+    "AttestationServiceCreationSpecificParams": {
```

Comment:
>AttestationServiceCreationSpecificParams [](start = 5, length = 40)

I noticed that policyName was removed in this API version. Is that intentional?

Reply:
Very intentional. policyName existed as a hack from 18 months ago when we were trying to go to private preview, but we did not yet have a final design for policy management. We now have a design for policy management APIs so the transition between RP and GA allows us the opportunity remove support for this hack (there are numerous horrible issues with the hack that I can go into in detail if you *really* care).